test/evaluation/limitDisclosureEvaluationHandler.spec.ts

```diff
@@ -16,20 +15,20 @@ describe('evaluate', () => {
 
   it('should return ok if verifiablePresentation doesn\'t have the etc field', function() {
     const pdSchema: PresentationDefinition = getFile('./test/dif_pe_examples/pd/pd-simple-schema-age-predicate.json').presentation_definition;
-    const vpSimple: Presentation = getFile('./test/dif_pe_examples/vp/vp-simple-age-predicate.json');
+    const vpSimple: VerifiablePresentation = getFile('./test/dif_pe_examples/vp/vp-simple-age-predicate.json');
     const evaluationClient: EvaluationClient = new EvaluationClient();
     vpSimple.holder = HOLDER_DID;
-    evaluationClient.evaluate(pdSchema, new VP(vpSimple));
-    expect(evaluationClient.verifiablePresentation.getVerifiableCredentials()[0]['etc']).toEqual(undefined);
+    evaluationClient.evaluate(pdSchema, vpSimple);
+    expect(evaluationClient.verifiablePresentation.verifiableCredential[0]['etc']).toEqual(undefined);
   });
 
   it('should return ok if verifiablePresentation doesn\'t have the birthPlace field', function() {
     const pdSchema: PresentationDefinition = getFile('./test/dif_pe_examples/pd/pd-schema-multiple-constraints.json').presentation_definition;
-    const vpSimple = getFile('./test/dif_pe_examples/vp/vp-multiple-constraints.json');
+    const vpSimple: VerifiablePresentation = getFile('./test/dif_pe_examples/vp/vp-multiple-constraints.json');
```

Comment:
we had to move these to TypeScript files from json files.

ps. all json example should be moved to ts.